scania-digital-design-system/sdds
# fix(event names): added event naming to the migrations docs

**Describe pull-request**  
Added a section in our migration docs on the event naming conventions. 

Note: Needs to add correct link to CODE_STYLES.md

**Solving issue**  
Fixes: [#](https://tegel.atlassian.net/browse/DTS-938)

**How to test**  
1. Go to storybook link below.
2. Check in migrations docs.
